### PR TITLE
spring-cloud-etcd-config documentation (issue #6 and #4)

### PR DESCRIPTION
**Pull request by lburgazzoli**
I've done a little bit of work about documentation, hope it helps

### CODE REVIEW

File: docs/src/main/asciidoc/spring-cloud-etcd.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Configuration is currently read on startup of the application.  Sending a HTTP POST to `/refresh` will cause the configuration to be reloaded.  Watching the key value store (which Etcd supports) is not currently possible, but will be a future addition to this project.
+
+=== How to activate
+
+Including a dependency on `org.springframework.cloud:spring-cloud-etcd-config` will enable auto-configuration that will setup Spring Cloud Etcd Config.
+
+=== Customizing
+
+Consul Config my be customized using the following properties:
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-etcd.adoc]: Consul :-)

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,175 @@
+// Do not edit this file (e.g. go instead to src/main/asciidoc)
+
+image::https://travis-ci.org/spring-cloud-incubator/spring-cloud-etcd.svg?branch=master[Build Status, link=https://travis-ci.org/spring-cloud-incubator/spring-cloud-etcd]
+
+This project provides Etcd integrations for Spring Boot apps through autoconfiguration
+and binding to the Spring Environment and other Spring programming model idioms. With a few
+simple annotations you can quickly enable and configure the common patterns inside your
+application and build large distributed systems with Etcd based components. The
+patterns provided include Service Discovery, Circuit Breaker and Configuration.
```
Comment (spencergibb) [inline README.adoc]: Should remove circuit breaker.

File: docs/src/main/asciidoc/intro.adoc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+This project provides Etcd integrations for Spring Boot apps through autoconfiguration
+and binding to the Spring Environment and other Spring programming model idioms. With a few
+simple annotations you can quickly enable and configure the common patterns inside your
+application and build large distributed systems with Etcd based components. The
+patterns provided include Service Discovery, Circuit Breaker and Configuration.
```
Comment (spencergibb) [inline docs/src/main/asciidoc/intro.adoc]: Should remove circuit breaker.

**Comment by spencergibb** [comment]
Make a couple small changes and it looks good.

**Comment by lburgazzoli** [comment]
Done

**Comment by spencergibb** [comment]
merged

**Pull request closed without merging**